cmd/console/main.go

```diff
@@ -62,11 +64,15 @@ func main() { // nolint:funlen,gocyclo
 		RunE: func(cmd *cobra.Command, args []string) error {
 			defer sentry.Flush(timeMult * time.Second)
 
-			store := storage.NewMemoryStorage()
 			if verbose {
 				consoleLogger = zerolog.ConsoleWriter{Out: os.Stdout}
 			}
 
+			if generateID {
+				_, err = fmt.Printf(uuid.New().String())
```

Comment:
change to 'fmt.Println(uuid.New().String())' to prevent '%' at end of line
and add 'os.Exit(0)' so that  stops execution

Reply:
added suggestions